### PR TITLE
Flag popo update locations

### PR DESCRIPTION
**Pull request by cvchaparro**
# Description

Include a summary of the changes and the related issue. List any dependencies that are required for this change.

# Linked Items:

Closes/Fixes/Resolves #XYZ

### Added

- _Describe any new features._

### Changed

- _Describe any changes in existing functionality._

### Deprecated

- _Describe any deprecated features._

### Removed

- _Describe any removed features._

### Fixed

- _Describe any bug fixes._

### Security

- _Describe any security-related changes._

# Checklist:

- [ ] I updated project documentation to reflect my changes.
- [ ] My changes generate no new warnings.
- [ ] I updated new and existing unit tests to account for my changes.
- [ ] I linked the associated item(s) to be closed.
- [ ] I bumped the version.
- [ ] I added the labels corresponding to my changes.

### CODE REVIEW

File: python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def validate_undefined_fields(
     def validate_dict(dict_to_validate: dict) -> None:
         # Due to a bug in how global validations are currently being applied, we need to look up the schema rather
         #   than rely on the schema provided by the validation system.
+        # POPO Update
```
Comment (lizzcondrey) [inline python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py:38]: Need to put the comment to the same line to remain consistent with the other flagging efforts.

**Review by crazynewidea — APPROVED** [review]
If you merge the latest popo-first-party-plugin-comments-pm branch, if might eliminate some of the flake8 errors.

**Comment by lizzcondrey** [comment]
> If you merge the latest popo-first-party-plugin-comments-pm branch, if might eliminate some of the flake8 errors.

That would be mixing work areas and task scopes. This is just flagging within validators.

File: python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py (hunk 1 above)
Comment (cvchaparro) [reply python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py:38]: I don't like end-of-line comments generally for the reason that they are harder to find without scrolling horizontally (for which I have a severe distaste) but for the sake of expediency I can update my comments.

File: python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ def validate_undefined_fields(
...
         definition_schema = get_definition_schema(definition_under_test, language_context)
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py:39]:
```suggestion
        definition_schema = get_definition_schema(definition_under_test, language_context)  # POPO Update
```

File: python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py [hunk 3]
```diff
@@ -47,6 +48,7 @@ def validate_dict(dict_to_validate: dict) -> None:
                     definition_under_test, undefined_field_message, VALIDATION_NAME, definition_under_test.get_lexeme_with_value(field_name)
                 )
 
+    # POPO Update
     dicts_to_test = get_substructures_by_type(definition_under_test, target_schema_definition, language_context)
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py:52]:
```suggestion
    dicts_to_test = get_substructures_by_type(definition_under_test, target_schema_definition, language_context)  # POPO Update
```

File: python/src/aac/plugins/validators/usecase_actions/_validate_usecase_actions.py [hunk 4]
```diff
@@ -38,6 +38,7 @@ def validate_usecase_actions(
     findings = ValidatorFindings()
 
     if definition_under_test.get_root_key() == ROOT_KEY_USECASE:
+        # POPO Update
         fields = definition_under_test.get_top_level_fields()
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/usecase_actions/_validate_usecase_actions.py:42]:
```suggestion
        fields = definition_under_test.get_top_level_fields()  # POPO Update
```

File: python/src/aac/plugins/validators/usecase_participants/_validate_usecase_participants.py [hunk 5]
```diff
@@ -36,6 +36,7 @@ def validate_usecase_participants(
     """
     findings = ValidatorFindings()
 
+    # POPO Update
     if definition_under_test.get_root_key() == ROOT_KEY_USECASE:
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/usecase_participants/_validate_usecase_participants.py:40]:
```suggestion
    if definition_under_test.get_root_key() == ROOT_KEY_USECASE:  # POPO Update
```

File: python/src/aac/plugins/validators/usecase_participants/_validate_usecase_participants.py [hunk 6]
```diff
@@ -49,6 +50,7 @@ def validate_usecase_participants(
 def _get_findings_from_invalid_step_endpoint(definition: Definition, endpoint_type: str) -> ValidatorFindings:
     findings = ValidatorFindings()
 
+    # POPO Update
     fields = definition.get_top_level_fields()
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/usecase_participants/_validate_usecase_participants.py:54]:
```suggestion
    fields = definition.get_top_level_fields()  # POPO Update
```

File: python/src/aac/plugins/validators/validator_implementation/_validate_validator_implementation.py [hunk 7]
```diff
@@ -31,6 +31,7 @@ def validate_validator_implementations(
     findings = ValidatorFindings()
 
     def validate_dict(dict_to_validate: dict) -> None:
+        # POPO Update
         validation_name = dict_to_validate.get(DEFINITION_FIELD_NAME, "")
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/validator_implementation/_validate_validator_implementation.py:35]:
```suggestion
        validation_name = dict_to_validate.get(DEFINITION_FIELD_NAME, "")  # POPO Update
```

File: python/src/aac/plugins/validators/validator_implementation/_validate_validator_implementation.py [hunk 8]
```diff
@@ -45,6 +46,7 @@ def validate_dict(dict_to_validate: dict) -> None:
             )
             logging.debug(no_implementation_error_message)
 
+    # POPO Update
     dicts_to_test = get_substructures_by_type(definition_under_test, target_schema_definition, language_context)
```
Comment (cvchaparro) [inline python/src/aac/plugins/validators/validator_implementation/_validate_validator_implementation.py:50]:
```suggestion
    dicts_to_test = get_substructures_by_type(definition_under_test, target_schema_definition, language_context)  # POPO Update
```

**Pull request merged**